protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.45.1 - unreleased
```

Comment:
You'll also need to bump the version in the manifest please.

Reply:
Also in the root manifest.

Reply:
Version bumped in `protocols/kad/Cargo.toml` and `Cargo.toml`. I'll let you resolve this conversation if it was what you meant.